src/js/providers/flash.js

```diff
@@ -125,11 +74,8 @@ define([
                 var level = levels[i];
                 // Set original index
                 level.index = i;
-                // Translate sources into quality levels, assigning custom levels if present
-                if (_customLabels && level.bitrate) {
-                    // get label with nearest rate match
-                    var sourceKbps = Math.round(level.bitrate / 1000);
-                    level.label = _getNearestCustomLabel(sourceKbps);
+                if (level.label !== 'Auto') {
+                    level.label = utils.generateLabel(level, _playerConfig.qualityLabels);
                 }
```

Comment:
Do we need to handle the case where two labels could end up having the same label if they are both closest to the same bandwidth?